### PR TITLE
Hooli-style Popup

### PR DESCRIPTION
**Pull request by AntonKottans**
### Hooli-style Popup

[Demo](https://antonkottans.github.io/hooli-style-popup/) |
[Code base](https://github.com/AntonKottans/hooli-style-popup)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/antonkottans/hooli-style-popup/style.css [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+
+.popup-item-img {
+	max-width: 48px;
+	max-height: 48px;
+	padding: 10px;
+	margin: auto 100%;
+	align-self: center;
+}
+
+a.popup-item:focus {
```
Comment (A-Ostrovnyy) [inline submissions/antonkottans/hooli-style-popup/style.css:114]: Avoid styling elements using html tag as selector. [Reason#1](https://frontstuff.io/you-need-to-stop-targeting-tags-in-css) [Reason#2](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity)
For base styles, it's ok but in main files/components, this is a bad practice.

File: submissions/antonkottans/hooli-style-popup/index.html [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+						<input
+							type="checkbox"
+							class="popup-checkbox hidden-checkbox"
+							id="popup-checkbox"
+						>
+						<label for="popup-checkbox" class="popup-checkbox-label">
+							<img class="nav-image" src="img/popup-button.png" alt="popup-menu" >
+						</label>
+						<div class="popup-menu">
+							<a href="#" class="popup-item"
```
Comment (A-Ostrovnyy) [inline submissions/antonkottans/hooli-style-popup/index.html:29]: Logically it looks like a list of links.

File: submissions/antonkottans/hooli-style-popup/index.html [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+								><img
+									class="popup-item-img"
+									alt="calendar"
```
Comment (A-Ostrovnyy) [inline submissions/antonkottans/hooli-style-popup/index.html:32]: For icons better set an empty `alt` attribute.
Sorry for russians but it's a good [article](https://htmlacademy.ru/blog/articles/alt-text)

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good Work!
See my comments below.

File: submissions/antonkottans/hooli-style-popup/index.html [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+                <li class="nav-menu-item">
+                    <a class="nav-menu-item-link" href="#">Mail</a>
+                </li>
+                <li class="nav-menu-item">
+                    <a class="nav-menu-item-link" href="#">Images</a>
+                </li>
+                <li class="nav-menu-item">
+                    <input type="checkbox" class="popup-checkbox hidden-checkbox" id="popup-checkbox">
+                    <label for="popup-checkbox" class="popup-checkbox-label">
+                        <img class="nav-image" src="img/popup-button.png" alt="popup-menu">
```
Comment (A-Ostrovnyy) [inline submissions/antonkottans/hooli-style-popup/index.html:24]: Why do you leave the populated `alt` attribute for this icon if you remove it from the rest icons?

File: submissions/antonkottans/hooli-style-popup/index.html [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+                <li class="nav-menu-item">
+                    <a class="nav-menu-item-link" href="#"><img class="nav-image" src="img/iconfinder_user_light.png"
+                            alt="user-img"></a>
+                </li>
+            </ul>
+        </nav>
+    </header>
+</body>
+
+</html>
```
Comment (A-Ostrovnyy) [inline submissions/antonkottans/hooli-style-popup/index.html:143]: Add empty lines at the end of files. [Reason](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline?noredirect=1&lq=1)

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good work!
Couple small issues needs to be done)

File: submissions/antonkottans/hooli-style-popup/index.html (hunk 4 above)
Comment (AntonKottans) [reply submissions/antonkottans/hooli-style-popup/index.html:24]: i did it via replace all, three img tags had different parameters order, sorry

File: submissions/antonkottans/hooli-style-popup/index.html (hunk 5 above)
Comment (AntonKottans) [reply submissions/antonkottans/hooli-style-popup/index.html:143]: and another stupid mistake, forgot that prettier deletes them

Comment (A-Ostrovnyy) [reply submissions/antonkottans/hooli-style-popup/index.html]: Check the settings of your code editor he can add an empty line on save automatically)

**Review by A-Ostrovnyy — APPROVED** [review]
Well done!

**Pull request merged**